platforms/hyperledger-indy/charts/indy-node/templates/configmap.yaml

```diff
@@ -10,4 +10,18 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
 data:
   indy_config.py: |  
-  {{ $.Values.configmap.indyConfig  | nindent 6 }}
\ No newline at end of file
+  {{ $.Values.configmap.indyConfig  | nindent 6 }}
+---
+apiVersion: v1
```

Comment:
should be separate chart - one configmap per org

Reply:
Its in separate chart now